cdap-ui/app/cdap/components/PipelineConfigurations/ConfigurationsContent/EngineConfigTabContent/index.js

```diff
@@ -69,29 +70,29 @@ class EngineConfigTabContent extends Component {
   }
 
   render() {
-    let pipelineTypeLabel = this.props.isBatch ? 'batch' : 'realtime';
+    let pipelineTypeLabel = GLOBALS.programLabel[this.props.pipelineType];
 
     return (
       <div
         id="engine-config-tab-content"
         className={classnames('configuration-step-content configuration-content-container', {
-          'batch-content': this.props.isBatch,
-          'realtime-content': !this.props.isBatch,
+          'batch-content': GLOBALS.etlBatchPipelines.indexOf(this.props.pipelineType) !== -1,
+          'realtime-content': GLOBALS.etlBatchPipelines.indexOf(this.props.pipelineType) === -1,
```

Comment:
Should be fine since SQL pipeline doesn't show the engine tab anyway. If/When we introduce different executors we can add specific CSS for SQL pipelines in later.